config/playground/helm/dev/dashboards.yaml

```diff
@@ -0,0 +1,203 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
```

Comment:
can you make comments here? why choose 1 replicaCount? thanks

Reply:
1 replicaCount was chosen for testing purposes. I have changed it to 3 now.